package.json

```diff
@@ -40,21 +40,23 @@
   "author": "St. Jude",
   "homepage": "http://stjudecloud.github.io/theme",
   "peerDependencies": {
-    "bootstrap": "4.3.1"
+    "bootstrap": "5.0.2"
   },
   "dependencies": {
-    "bootstrap": "4.3.1",
-    "clean-css-cli": "^5.3.2",
-    "node-sass": "^6.0.1",
-    "node-sass-package-importer": "^5.3.2",
-    "nodemon": "^2.0.12",
+    "bootstrap": "5.0.2",
+    "clean-css-cli": "^4.3.0",
+    "node-sass-package-importer": "^5.3.1",
+    "nodemon": "^1.19.4",
```

Comment:
Was downgrading these intentional?

Also, do we still need `node-sass-package-importer` since we are moving off of `node-sass`?

Reply:
Thanks, no, just `sass-loader`.